storybook/qmlTests/tests/tst_OnboardingLayout.qml

```diff
@@ -896,7 +929,7 @@ Item {
             verify(!!resultData)
             compare(resultData.password, "")
             compare(resultData.enableBiometrics, data.biometrics && data.bioEnabled)
-            compare(resultData.keycardPin, mockDriver.existingPin)
+            compare(resultData.keycardPin, "")
```

Comment:
This changes the expectation of the test... :thinking:

Reply:
But this test has nothing to do with the PIN, right?

Reply:
Not 100% sure now, maybe the PIN is not exactly needed in this flow? The purpose of this test check is to ensure we get back the PIN that we collected thru the onboarding

Reply:
Alright, I think I figured it out.

in `Log in with Keycard`, we don't need to _collect_ the PIN. We just enter it once during authorization and forget it. That's why we compare it with empty.

In face, I think there are no cases when we need to _collect_ the PIN. We always use it right away to authorize, and don't go further if auth fails.